packages/data-transport-layer/src/services/l2-ingestion/handlers/transaction.ts

```diff
@@ -110,6 +110,18 @@ export const handleSequencerBlock = {
     },
     db: TransportDB
   ): Promise<void> => {
+    // Defend against situations where we somehow miss a transaction during sync
+    if (entry.transactionEntry.index > 0) {
```

Comment:
This is just checking that the entry has any properties, yeah? Can we assert it has the properties we intend to use as well?

Reply:
This is just checking that it's not the very first transaction entry. I didn't add any property assertions here (I probably should).